checks/dependency_update_tool.go

```diff
@@ -25,11 +25,11 @@ const CheckDependencyUpdateTool = "Dependency-Update-Tool"
 
 //nolint
 func init() {
-	registerCheck(CheckDependencyUpdateTool, AutomaticDependencyUpdate)
+	registerCheck(CheckDependencyUpdateTool, DependencyUpdateTool)
 }
 
-// AutomaticDependencyUpdate will check the repository if it contains Automatic dependency update.
-func AutomaticDependencyUpdate(c *checker.CheckRequest) checker.CheckResult {
+// DependencyUpdateTool will check the repository uses a dependency update tool.
+func DependencyUpdateTool(c *checker.CheckRequest) checker.CheckResult {
```

Comment:
Nit: rename to `HasDependencyUpdateTool`

Reply:
renamed to `UsesDependencyUpdateTool`. Ok or `Has` sounds better?

Reply:
SGTM